api/censuses.go

```diff
@@ -201,7 +201,7 @@ func (a *API) censusAddHandler(msg *apirest.APIdata, ctx *httprouter.HTTPContext
 		if p.Weight == nil {
 			p.Weight = new(types.BigInt).SetUint64(1)
 		}
-		if ref.Indexed && p.Weight.ToInt().Uint64() != 1 {
+		if ref.Indexed && p.Weight.ToStdBigInt().Uint64() != 1 {
```

Comment:
What's the point for replacing "ToInt()"  to "ToStdBigInt" ?

Reply:
When I was coding this PR, specifically the part touching the abi.Encode I needed a method to cast results from types.BigInt to math/big.
When I looked at the code at first glance I did not find anything and later on, I found that ToInt()  what it actually does is to cast to math/big. This is, at least, anti-intuitive and I cannot find a single reason on why someone should expect that a method called ToInt() instead of casting a bigInt to Int cast from types.BigInt to a math/big BigInt.

So, I've decided to change the name to a more suited one.

Reply:
I see. So `ToInt()` method name did not provide enough info about what it actually does. However if we are changing this, I'd not use`ToStdBigInt`, taking ideas from other big.Int wrappers, a better naming IMO would be `MathBigInt()` which clearly indicates what is the method returning.

Reply:
I agree that MathBigInt() is a better naming. 
When I'm searching for methods, my brain would expect ToMathBigInt() but is just opinionated I guess, the next thing I would do is to search without the To keyword so I guess MathBigInt() is fine too :)
Thank you for the suggestion on the naming.

Reply:
Changed :)